### ISSUE TITLE
Cluster dissapears on max zoom out

### ISSUE THREAD

**Issue by GoogleCodeExporter** [seancallinan/google-maps-utility-library-v3#156]
```
Demo link or sample code:
http://goo.gl/oiklq

What steps will reproduce the problem?
1.Load page
2.Click to zoom out
3.cluster dissapears

Expected result:
I expected the cluster to stay visible

Actual result:
The cluster dissapears until you zoom in again or move the map.

Version: ###

Browser / Operating System:
Google Chrome

Additional comments:
I tried like v=3.5 v=3.6 and v=3.7


*********************************************************
Tip: Star this issue (next to title) to receive notifications of status
changes against this issue, also used as a gauge for how many people are
interested in seeing it resolved.
*********************************************************

```

Original issue reported on code.google.com by `emrulla...@gmail.com` on 30 Jan 2012 at 11:16

**Comment by GoogleCodeExporter** [comment]
```
This behavior is caused by a Google bug: when a map is at level 0 and "-" of 
zoom slider is clicked, a "zoom_changed" event is fired even though the map 
doesn't zoom out any further. In this situation, no "idle" event is triggered 
as usually happens when the map actually zooms.

MarkerClustererPlus removes all clusters in the zoom_changed handler, then 
redraws them when the subsequent idle event is handled. Since the latter never 
occurs when the map is at level 0, the clusters disappear.

This will be fixed in the next update to MarkerClustererPlus.

```

Original comment by `garylitt...@gmail.com` on 21 Feb 2012 at 7:22
- Changed state: **Confirmed**

**Comment by GoogleCodeExporter** [comment]
```
There is a fix for this in the current trunk release (2.0.9).
```

Original comment by `garylitt...@gmail.com` on 21 Feb 2012 at 7:35
- Changed state: **FixedNotReleased**

**Comment by GoogleCodeExporter** [comment]
Original comment by `garylitt...@gmail.com` on 22 Feb 2012 at 5:31
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Using version 2.0.9, it appears that this is fixed at zoom level 0, but still a 
problem at max zoom.  E.g., if I don't have a maxZoom set, MarkerClusterer will 
create clusters when the map is zoomed all the way in, which is what I want.  
But if I zoom in, the cluster disappears.  The attached patch fixes the issue 
for me.
```

Original comment by `eric.ent...@gmail.com` on 1 Mar 2012 at 1:14

Attachments:
- [max_zoom_workaround.diff](https://storage.googleapis.com/google-code-attachments/google-maps-utility-library-v3/issue-156/comment-4/max_zoom_workaround.diff)

**Comment by GoogleCodeExporter** [comment]
```
Fixed in the trunk (2.0.10). Please verify and I will tag the release.
```

Original comment by `garylitt...@gmail.com` on 1 Mar 2012 at 1:50
- Changed state: **FixedNotReleased**

**Comment by GoogleCodeExporter** [comment]
```
For me with 2.0.10 the cluster still seems to dissapear.
```

Original comment by `emrulla...@gmail.com` on 1 Mar 2012 at 8:23

**Comment by GoogleCodeExporter** [comment]
```
Sorry for the slow response - 2.0.10 fixes the issue for me.
```

Original comment by `eric.ent...@gmail.com` on 13 Mar 2012 at 1:16

**Comment by GoogleCodeExporter** [comment]
```
Fixed in tagged release (2.0.10)
```

Original comment by `garylitt...@gmail.com` on 13 Mar 2012 at 1:42
- Changed state: **Fixed**
